keyboards/hineybush/hbcp/info.json

```
@@ -0,0 +1,15 @@
+        },
+        "LAYOUT_WKL": {
```

I missed this before.

```suggestion
        "LAYOUT_wkl": {
```